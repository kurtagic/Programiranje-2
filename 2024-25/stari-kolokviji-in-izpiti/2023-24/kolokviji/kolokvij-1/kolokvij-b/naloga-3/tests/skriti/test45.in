52 34 461
324 208 900 163 276 395 -34 -954 -738 -826 -623 423 -814 811 255 130 787 -894 464 -168 720 445 210 -466 43 970 -238 492 -362 -355 78 590 -796 854 -882 -398 -984 -324 236 425 71 669 721 682 664 760 -12 331 297 -598 616 124 -216 707 383 -302 56 467 -557 -870 167 -503 -97 79 232 184 118 -717 -151 972 778 -402 -874 11 -665 -671 -114 -566 327 824 -12 -553 978 45 721 967 -150 207 651 393 -683 897 14 151 442 902 -295 -905 253 -166 949 -686 -297 -837 -232 -816 21 -197 -629 230 -96 930 -583 425 -649 228 -115 840 -127 428 219 849 513 495 -980 431 712 -483 750 -860 371 50 -797 446 -76 762 448 -802 51 528 -411 -198 -254 157 -235 -43 430 309 -59 723 -909 559 569 -689 262 356 -944 -524 1 -525 -138 92 -729 -989 -171 998 -387 733 481 -28 946 301 -833 833 35 -10 -341 -458 -268 -23 585 -139 -669 -429 835 343 26 -305 560 -93 727 -769 -865 -391 64 -498 -809 -714 -552 -639 -943 712 309 -284 810 -986 693 -226 354 454 681 642 149 -975 352 -417 -345 961 -637 -432 -871 -97 881 -51 997 -210 -817 -790 -370 -642 -998 87 -278 -679 -911 938 675 -656 206 81 302 414 -854 -784 189 -338 718 -855 182 554 166 59 77 -715 391 123 385 -554 -750 -729 62 730 -945 -877 733 929 265 4 231 470 906 -224 -510 -93 966 176 34 -898 254 -586 -658 205 -933 584 179 -844 12 -189 883 -895 778 384 654 354 -433 130 -502 437 -257 -742 -466 -495 681 457 644 -307 -36 -859 457 218 702 -733 551 19 702 -523 -535 78 -500 -534 634 36 -209 573 -776 470 307 -539 225 30 -65 885 -622 -470 -728 950 410 632 503 -461 -268 452 -68 -703 -351 -205 -84 -569 -330 -656 -927 573 723 453 263 -539 766 378 -701 -564 -960 940 -218 121 389 -333 170 42 500 -490 396 473 726 -908 147 -160 535 344 -817 764 -134 -118 -947 648 -202 -725 -979 -468 -269 93 54 32 -633 -615 307 139 -631 712 -230 255 837 946 -931 -436 -388 -205 -48 -370 -803 -96 148 343 992 419 -127 647 577 -556 935 -188 -792 52 590 -160 120 94 332 -244 -718 -205 552 -444 -540 910 851 -220 75 836 983 -996 237 465 -134 224 -483 -836 -378 800 -641 -321 -92 -945 -87 864 -497 488 -327 -699 666 893 -456
0 11 20 32 43 52 61 67 75 88 96 105 114 126 137 148 152 160 170 184 196 203 211 217 221 230 243 254 261 271 278 282 288 295 303 310 322 327 334 343 353 360 368 380 392 402 419 426 430 437 445 453 461
1 2 5 9 11 12 13 21 23 28 31 1 6 8 16 18 19 23 24 28 3 5 6 7 11 16 19 20 21 25 30 32 3 14 16 22 24 26 29 30 31 32 33 3 9 11 23 25 26 27 28 30 1 5 12 13 16 20 29 31 33 1 12 14 18 24 26 0 5 6 7 12 14 22 29 1 3 4 6 12 14 16 17 20 24 27 28 31 0 1 4 12 14 16 20 24 4 8 13 17 22 24 26 27 30 1 11 15 18 22 24 29 32 33 0 5 6 7 8 9 10 12 16 18 28 33 0 7 12 18 20 22 23 25 26 27 29 0 1 2 4 7 10 20 21 22 23 32 12 22 24 25 2 4 5 9 19 23 26 31 4 11 12 18 23 25 26 27 30 32 1 4 7 8 9 11 13 15 17 21 23 24 25 32 0 4 7 9 10 12 13 20 25 27 30 33 2 7 17 23 27 30 32 6 9 11 17 18 23 26 33 8 15 16 20 26 30 5 9 21 23 4 8 10 12 18 29 31 32 33 4 5 6 8 11 13 14 15 16 17 25 29 31 0 3 6 8 12 14 18 23 24 28 31 13 14 15 25 28 29 30 0 7 9 14 17 18 20 22 24 33 0 8 15 22 24 25 31 5 14 25 31 3 4 8 11 13 28 0 2 17 18 23 30 32 3 12 15 17 20 22 27 29 1 19 20 22 23 24 31 1 2 3 4 6 13 14 19 20 21 24 33 5 15 18 26 33 4 12 14 17 22 24 32 2 5 6 14 15 19 23 24 27 2 4 6 8 11 12 14 16 24 27 3 5 6 16 22 25 28 4 7 16 20 24 25 29 31 2 3 6 11 14 19 20 21 22 23 25 32 2 3 6 7 10 13 15 17 22 24 27 33 2 8 11 12 15 19 20 21 25 27 0 2 4 5 9 10 15 16 17 18 22 25 26 27 29 32 33 0 5 7 12 24 29 32 3 11 20 32 0 2 6 14 18 25 29 2 6 7 8 13 29 31 33 8 11 16 20 23 24 26 31 5 12 13 21 22 23 27 30
