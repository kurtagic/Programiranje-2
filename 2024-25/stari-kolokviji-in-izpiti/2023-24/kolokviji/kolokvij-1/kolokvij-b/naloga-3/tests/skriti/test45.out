0_324_208_0_0_900_0_0_0_163_0_276_395_-34_0_0_0_0_0_0_0_-954_0_-738_0_0_0_0_-826_0_0_-623_0_0
0_423_0_0_0_0_-814_0_811_0_0_0_0_0_0_0_255_0_130_787_0_0_0_-894_464_0_0_0_-168_0_0_0_0_0
0_0_0_720_0_445_210_-466_0_0_0_43_0_0_0_0_970_0_0_-238_492_-362_0_0_0_-355_0_0_0_0_78_0_590_0
0_0_0_-796_0_0_0_0_0_0_0_0_0_0_854_0_-882_0_0_0_0_0_-398_0_-984_0_-324_0_0_236_425_71_669_721
0_0_0_682_0_0_0_0_0_664_0_760_0_0_0_0_0_0_0_0_0_0_0_-12_0_331_297_-598_616_0_124_0_0_0
0_-216_0_0_0_707_0_0_0_0_0_0_383_-302_0_0_56_0_0_0_467_0_0_0_0_0_0_0_0_-557_0_-870_0_167
0_-503_0_0_0_0_0_0_0_0_0_0_-97_0_79_0_0_0_232_0_0_0_0_0_184_0_118_0_0_0_0_0_0_0
-717_0_0_0_0_-151_972_778_0_0_0_0_-402_0_-874_0_0_0_0_0_0_0_11_0_0_0_0_0_0_-665_0_0_0_0
0_-671_0_-114_-566_0_327_0_0_0_0_0_824_0_-12_0_-553_978_0_0_45_0_0_0_721_0_0_967_-150_0_0_207_0_0
651_393_0_0_-683_0_0_0_0_0_0_0_897_0_14_0_151_0_0_0_442_0_0_0_902_0_0_0_0_0_0_0_0_0
0_0_0_0_-295_0_0_0_-905_0_0_0_0_253_0_0_0_-166_0_0_0_0_949_0_-686_0_-297_-837_0_0_-232_0_0_0
0_-816_0_0_0_0_0_0_0_0_0_21_0_0_0_-197_0_0_-629_0_0_0_230_0_-96_0_0_0_0_930_0_0_-583_425
-649_0_0_0_0_228_-115_840_-127_428_219_0_849_0_0_0_513_0_495_0_0_0_0_0_0_0_0_0_-980_0_0_0_0_431
712_0_0_0_0_0_0_-483_0_0_0_0_750_0_0_0_0_0_-860_0_371_0_50_-797_0_446_-76_762_0_448_0_0_0_0
-802_51_528_0_-411_0_0_-198_0_0_-254_0_0_0_0_0_0_0_0_0_157_-235_-43_430_0_0_0_0_0_0_0_0_309_0
0_0_0_0_0_0_0_0_0_0_0_0_-59_0_0_0_0_0_0_0_0_0_723_0_-909_559_0_0_0_0_0_0_0_0
0_0_569_0_-689_262_0_0_0_356_0_0_0_0_0_0_0_0_0_-944_0_0_0_-524_0_0_1_0_0_0_0_-525_0_0
0_0_0_0_-138_0_0_0_0_0_0_92_-729_0_0_0_0_0_-989_0_0_0_0_-171_0_998_-387_733_0_0_481_0_-28_0
0_946_0_0_301_0_0_-833_833_35_0_-10_0_-341_0_-458_0_-268_0_0_0_-23_0_585_-139_-669_0_0_0_0_0_0_-429_0
835_0_0_0_343_0_0_26_0_-305_560_0_-93_727_0_0_0_0_0_0_-769_0_0_0_0_-865_0_-391_0_0_64_0_0_-498
0_0_-809_0_0_0_0_-714_0_0_0_0_0_0_0_0_0_-552_0_0_0_0_0_-639_0_0_0_-943_0_0_712_0_309_0
0_0_0_0_0_0_-284_0_0_810_0_-986_0_0_0_0_0_693_-226_0_0_0_0_354_0_0_454_0_0_0_0_0_0_681
0_0_0_0_0_0_0_0_642_0_0_0_0_0_0_149_-975_0_0_0_352_0_0_0_0_0_-417_0_0_0_-345_0_0_0
0_0_0_0_0_961_0_0_0_-637_0_0_0_0_0_0_0_0_0_0_0_-432_0_-871_0_0_0_0_0_0_0_0_0_0
0_0_0_0_-97_0_0_0_881_0_-51_0_997_0_0_0_0_0_-210_0_0_0_0_0_0_0_0_0_0_-817_0_-790_-370_-642
0_0_0_0_-998_87_-278_0_-679_0_0_-911_0_938_675_-656_206_81_0_0_0_0_0_0_0_302_0_0_0_414_0_-854_0_0
-784_0_0_189_0_0_-338_0_718_0_0_0_-855_0_182_0_0_0_554_0_0_0_0_166_59_0_0_0_77_0_0_-715_0_0
0_0_0_0_0_0_0_0_0_0_0_0_0_391_123_385_0_0_0_0_0_0_0_0_0_-554_0_0_-750_-729_62_0_0_0
730_0_0_0_0_0_0_-945_0_-877_0_0_0_0_733_0_0_929_265_0_4_0_231_0_470_0_0_0_0_0_0_0_0_906
-224_0_0_0_0_0_0_0_-510_0_0_0_0_0_0_-93_0_0_0_0_0_0_966_0_176_34_0_0_0_0_0_-898_0_0
0_0_0_0_0_254_0_0_0_0_0_0_0_0_-586_0_0_0_0_0_0_0_0_0_0_-658_0_0_0_0_0_205_0_0
0_0_0_-933_584_0_0_0_179_0_0_-844_0_12_0_0_0_0_0_0_0_0_0_0_0_0_0_0_-189_0_0_0_0_0
883_0_-895_0_0_0_0_0_0_0_0_0_0_0_0_0_0_778_384_0_0_0_0_654_0_0_0_0_0_0_354_0_-433_0
0_0_0_130_0_0_0_0_0_0_0_0_-502_0_0_437_0_-257_0_0_-742_0_-466_0_0_0_0_-495_0_681_0_0_0_0
0_457_0_0_0_0_0_0_0_0_0_0_0_0_0_0_0_0_0_644_-307_0_-36_-859_457_0_0_0_0_0_0_218_0_0
0_702_-733_551_19_0_702_0_0_0_0_0_0_-523_-535_0_0_0_0_78_-500_-534_0_0_634_0_0_0_0_0_0_0_0_36
0_0_0_0_0_-209_0_0_0_0_0_0_0_0_0_573_0_0_-776_0_0_0_0_0_0_0_470_0_0_0_0_0_0_307
0_0_0_0_-539_0_0_0_0_0_0_0_225_0_30_0_0_-65_0_0_0_0_885_0_-622_0_0_0_0_0_0_0_-470_0
0_0_-728_0_0_950_410_0_0_0_0_0_0_0_632_503_0_0_0_-461_0_0_0_-268_452_0_0_-68_0_0_0_0_0_0
0_0_-703_0_-351_0_-205_0_-84_0_0_-569_-330_0_-656_0_-927_0_0_0_0_0_0_0_573_0_0_723_0_0_0_0_0_0
0_0_0_453_0_263_-539_0_0_0_0_0_0_0_0_0_766_0_0_0_0_0_378_0_0_-701_0_0_-564_0_0_0_0_0
0_0_0_0_-960_0_0_940_0_0_0_0_0_0_0_0_-218_0_0_0_121_0_0_0_389_-333_0_0_0_170_0_42_0_0
0_0_500_-490_0_0_396_0_0_0_0_473_0_0_726_0_0_0_0_-908_147_-160_535_344_0_-817_0_0_0_0_0_0_764_0
0_0_-134_-118_0_0_-947_648_0_0_-202_0_0_-725_0_-979_0_-468_0_0_0_0_-269_0_93_0_0_54_0_0_0_0_0_32
0_0_-633_0_0_0_0_0_-615_0_0_307_139_0_0_-631_0_0_0_712_-230_255_0_0_0_837_0_946_0_0_0_0_0_0
-931_0_-436_0_-388_-205_0_0_0_-48_-370_0_0_0_0_-803_-96_148_343_0_0_0_992_0_0_419_-127_647_0_577_0_0_-556_935
-188_0_0_0_0_-792_0_52_0_0_0_0_590_0_0_0_0_0_0_0_0_0_0_0_-160_0_0_0_0_120_0_0_94_0
0_0_0_332_0_0_0_0_0_0_0_-244_0_0_0_0_0_0_0_0_-718_0_0_0_0_0_0_0_0_0_0_0_-205_0
552_0_-444_0_0_0_-540_0_0_0_0_0_0_0_910_0_0_0_851_0_0_0_0_0_0_-220_0_0_0_75_0_0_0_0
0_0_836_0_0_0_983_-996_237_0_0_0_0_465_0_0_0_0_0_0_0_0_0_0_0_0_0_0_0_-134_0_224_0_-483
0_0_0_0_0_0_0_0_-836_0_0_-378_0_0_0_0_800_0_0_0_-641_0_0_-321_-92_0_-945_0_0_0_0_-87_0_0
0_0_0_0_0_864_0_0_0_0_0_0_-497_488_0_0_0_0_0_0_0_-327_-699_666_0_0_0_893_0_0_-456_0_0_0
